pkg/setup/setup.go

```diff
@@ -285,12 +289,22 @@ func (s *Setup) installDefaultPlugins(httpClient *httpclient.Client) error {
 
 	// Install dcos-core-cli.
 	errCore := s.installPlugin("dcos-core-cli", httpClient)
+
 	// The installation of the core and EE plugins happen in parallel.
 	// We wait for the installation of the enterprise plugin before returning.
 	errEnterprise := <-enterpriseInstallErr
 
 	if errCore != nil {
-		return errCore
+		// check that the version is 1.12 and if so, try to install the bundled plugin
+		if versionNumber(version.Version) == "1.12" {
+			err = s.installBundledPlugin()
+			if err != nil {
+				// We don't return an error as the EE plugin is not as useful as the core plugin.
+				return fmt.Errorf("unable to install DC/OS core CLI plugin: %s", err)
```

Comment:
We use normal sentences for log messages, eg. `Unable to install DC/OS core CLI plugin.`

Reply:
That isn't a log message though, it's an error message where you aren't supposed to write a normal sentence.